indico/modules/events/templates/admin/autolinker.html

```diff
@@ -0,0 +1,41 @@
+{% extends 'layout/admin_page.html' %}
+{% from 'events/admin/_event_label_list.html' import render_event_label_list %}
+
+{% block title %}
+    {% trans %}Auto-linked Patterns{% endtrans %}
+{% endblock %}
+
+{% block description %}
+    <div class="ui icon purple message">
+        <i class="lab icon"></i>
+        <div class="content">
+            <div class="header">{% trans %}Experimental feature{% endtrans %}</div>
```

Comment:
hmm this message will probably stay forever? :D

Reply:
🧪  Naaaaahhh